cub/cub/device/dispatch/kernels/kernel_reduce.cuh

```diff
@@ -335,12 +335,12 @@ __launch_bounds__(int(ChainedPolicyT::ActivePolicy::ReducePolicy::BLOCK_THREADS)
   int n_threads = reduce_grid_size * block_threads;
 
   _CCCL_PRAGMA_UNROLL_FULL()
-  for (::cuda::std::uint32_t i = tid; i < num_items; i += (n_threads * items_per_thread))
+  for (int i = tid; i < num_items; i += (n_threads * items_per_thread))
   {
     ftype items[items_per_thread] = {};
     for (int j = 0; j < items_per_thread; j++)
     {
-      const ::cuda::std::uint32_t idx = i + j * n_threads;
+      const int idx = i + j * n_threads;
```

Comment:
@srinivasyadav18 you should take a look at this

Reply:
The change looks fine to me.